tests/spectron/spectronWebActions.js

```diff
@@ -351,6 +370,38 @@ class WebActions {
         let elements = await this.app.client.elements(locator);
         return elements.value.length;
     }
+    
+    async navigateURL(url) {
+        return this.app.client.url(url)
+    }
+
+    async verifySpellCheckerWorking(locator, previous) {
+        let current = await this.getValue(locator);
+        await expect(current !== previous).toBeTruthy();
+    }
+
+    async openContextMenu(locator) {
+        if (isWindowsOS) {
```

Comment:
Do we need a different logic for Windows?

Reply:
Yes, on Windows I have to use robot.js to perform a right click. A right click on selenium didn't bring to a correct selected value for textbox